docs/development/setup_development.rst

```diff
@@ -103,7 +103,7 @@ created by default when running ``make dev``. In addition, sources and
 submissions are present. The test users have the following credentials:
 
 * **Username:** ``journalist`` or ``dellsberg``
-* **Password:** ``WEjwn8ZyczDhQSK24YKM8C9a``
```

Comment:
If we really want to keep it, I can do that, but it would mean adding (a small amount) of complexity to get code. It's already very terse, and we'd basically end up factoring out a single function call to get this behavior.

Reply:
I am in favor of your change.